index.html

```diff
@@ -157,10 +157,28 @@ <h5 class="about-heading">About Me</h5>
                     </ul>
                 </li>
             </ul>
-            
         </section>
-             
     </section>
-
+    <footer >
+    <section class="form">
+        <h5 class="form-heading">I'm always<br>interested in hearing about new projects,
+         so if you'd like to chat please get in touch.</h5>
+        <form  class="form-inputs" action="https://formspree.io/f/xeqdvepo" method="post">
+        <input class="input" type="text" minlength="30" placeholder="Enter full name" required>
+        <input  class="input" type="email" placeholder="Enter Email" required>
+        <textarea name="comment" id="comment" maxlength="500">Enter text here.</textarea>
```

Comment:
- [ ] Kindly ensure the user cannot submit the form without typing the message.
`Hint: You may consider add the required attribute on the textarea`